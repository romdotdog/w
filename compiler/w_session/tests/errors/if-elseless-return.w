fn main(a: u32, b: u32): u32 {
    if a {
        b
    }
}