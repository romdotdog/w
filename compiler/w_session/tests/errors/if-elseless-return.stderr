tests/errors/if-elseless-return.w:5:1: error: branch must return void
}
^
